eng/mgmt/automation/typespec_utils.py

```diff
@@ -0,0 +1,67 @@
+import os
+import re
+import sys
+import yaml
+import logging
+
+
+def validate_tspconfig(tsp_dir: str) -> bool:
+    valid = True
+
+    if not tsp_dir.endswith('tspconfig.yaml'):
+        tsp_dir = os.path.join(tsp_dir, 'tspconfig.yaml')
+    with open(tsp_dir, 'r') as tspconfig:
```

Comment:
Do we need to handle/log an error when we fail to open the file?

Reply:
In current agreement, upstream toolchain (the SDK automation framework) should all necessary files (and properties -- which include those `service-dir` etc.) exists, before invoking Java automation on TypeSpec source.